features/step_definitions/HomePageStep_SignedIn.spec.js

```diff
@@ -1,61 +1,66 @@
 const{Given, When, Then} = require('@cucumber/cucumber');
 const{expect} = require('@playwright/test');
 const {POManager} = require('../../pageObjects/POManager');
+const constants = require('../../utils/appConstants.spec.js');
+
+
 
 When('User clicks on Get Started button of Data Structures-Introduction module', async function () {
     await this.homePage.clickGetStartedOf_DataStructure();
   });
 
   Then('User is able to go to Data Structures-Introduction page', async function () {
-    await expect(this.page).toHaveTitle("Data Structures-Introduction");
+    await expect(this.page).toHaveTitle(constants.DS_PAGE_TITLE);
+    //await expect(this.page).toHaveTitle("Data Structures-Introduction");
+
   });
 
   When('User clicks on Get Started button of Array module', async function () {
     await this.homePage.clickGetStartedOf_Array();
   });
 
   Then('User is able to go to Array page', async function () {
-    await expect(this.page).toHaveTitle("Array");
+    await expect(this.page).toHaveTitle(constants.ARRAY_PAGE_TITLE);
```

Comment:
Can we use [cucumber expression](https://github.com/cucumber/cucumber-expressions#readme) here? get the value from scenario only instead of appConstants?
Example:
Scenario: Verify user can navigate to Array page from Home Page
	When User clicks on Get Started button of Array module
   	Then User is able to go to "Array" page

Then('User is able to go to {String} page', async function (pagename) {
    await expect(this.page).toHaveTitle(pagename);

That will reduce your steps from 5 steps to 1 step